### PR TITLE
remove unneeded variable & proper element creation

### PR DESCRIPTION
**Pull request by MorrisJobke**
cc @nickvergessen @LukasReschke @rullzer 

Found while checking downstreams.

### CODE REVIEW

**Comment by MorrisJobke** [comment]
@nickvergessen I also added the proper creation of the element.

File: apps/updatenotification/js/notification.js [hunk 1]
```diff
@@ -14,11 +14,10 @@
  * this gets only loaded if an update is available and then shows a temporary notification
  */
 $(document).ready(function(){
-	var head = $('html > head'),
-		version = oc_updateState.updateVersion,
+	var version = oc_updateState.updateVersion,
 		docLink = oc_updateState.updateLink,
```
Comment (MorrisJobke) [inline apps/updatenotification/js/notification.js]: You mean that I should use them directly? Sure :)

**Comment by MorrisJobke** [comment]
@nickvergessen I addressed your comment ;)

**Comment by nickvergessen** [comment]
👍

**Comment by rullzer** [comment]
LGTM

**Review by rullzer — APPROVED** [review]

**Pull request merged**